src/common/com/intellij/plugins/haxe/ide/quickfix/CreateGetterSetterQuickfix.java

```diff
@@ -0,0 +1,90 @@
+package com.intellij.plugins.haxe.ide.quickfix;
+
+import com.intellij.codeInsight.intention.impl.BaseIntentionAction;
+import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.openapi.command.WriteCommandAction;
+import com.intellij.openapi.editor.Editor;
+import com.intellij.openapi.project.Project;
+import com.intellij.plugins.haxe.ide.generation.GetterSetterMethodBuilder;
+import com.intellij.plugins.haxe.lang.psi.HaxeNamedComponent;
+import com.intellij.plugins.haxe.model.HaxeClassModel;
+import com.intellij.plugins.haxe.model.HaxeFieldModel;
+import com.intellij.plugins.haxe.util.HaxeElementGenerator;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiFile;
+import com.intellij.psi.PsiParserFacade;
+import com.intellij.util.IncorrectOperationException;
+import org.jetbrains.annotations.Nls;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+/**
+ * @author Ilya Malanin
+ */
+
+public class CreateGetterSetterQuickfix extends BaseIntentionAction {
+  private final HaxeClassModel classModel;
+  private final HaxeFieldModel field;
+  private final Boolean generateGetter;
+
+  public CreateGetterSetterQuickfix(HaxeClassModel classModel, HaxeFieldModel field, Boolean generateGetter) {
+    this.classModel = classModel;
+    this.field = field;
+    this.generateGetter = generateGetter;
+  }
+
+  @NotNull
+  @Override
+  public String getText() {
+    return generateGetter ? "Generate getter" : "Generate setter";
+  }
+
+  @Nls
+  @NotNull
+  @Override
+  public String getFamilyName() {
+    return getText();
+  }
+
+  @Override
+  public boolean isAvailable(@NotNull Project project, Editor editor, PsiFile file) {
+    return true;
```

Comment:
And, yes, I've talked about more intelligent getter/setter generators with Slava Ra and he gave me a lot on information to think about. 

e.g.  for code:
```haxe
public var variable(get, set):Int;
```

We should register two proposals if one of method is generated (setter or getter):
- Generate getter/setter
- Generate getter/setter and mark property as @:isVar

Second proposal is needed if user isn't going to set or get value to/from an another source, and then code is become valid.

Not valid code:
```haxe
public var variable(get, set):Int;

function set_variable(value:Int):Int {
   return variable = value;
}

function get_variable():Int {
   return variable;
}
```

Compilation error: 
```
This field cannot be accessed because it is not a real variable
Add @:isVar here to enable it
```

Valid code:
```haxe
@:isVar public var variable(get, set):Int;

function set_variable(value:Int):Int {
   return variable = value;
}

function get_variable():Int {
   return variable;
}
```

And so on, there is a lot of things to improve, but let's do it gradually.